doc/cli/cli_show.rst

```diff
@@ -0,0 +1,130 @@
+    .
+    .
+    .
+    (0043, 104b) [DAS xm pattern]                    SL: 0
+    (0043, 104c) [TGGC trigger mode]                 SS: 0
+    (0043, 104d) [Start scan to X-ray on delay]      FL: 0.0
+    (0043, 104e) [Duration of X-ray on]              FL: 10.60060977935791
+    (7fe0, 0010) Pixel Data                          OW: Array of 32768 elements
+    (fffc, fffc) Data Set Trailing Padding           OB: Array of 126 elements
+
+Note that prefixing the file specification with "pydicom::" will read the file
+from the *pydicom* test data files rather than from the file system.
+
+You can also show just parts of the DICOM file by specifying a data element
+using the usual pydicom keyword notation:
+
+.. code-block:: console
+
+    $ pydicom show pydicom::CT_small.dcm::PatientName
+    CompressedSamples^CT1
+
+    $ pydicom show pydicom::rtplan.dcm::FractionGroupSequence
+    [(300a, 0071) Fraction Group Number               IS: "1"
+    (300a, 0078) Number of Fractions Planned         IS: "30"
+    (300a, 0080) Number of Beams                     IS: "1"
+    (300a, 00a0) Number of Brachy Application Setups IS: "0"
+    (300c, 0004)  Referenced Beam Sequence  1 item(s) ----
+    (300a, 0082) Beam Dose Specification Point       DS: [239.531250000000, 239.531250000000, -751.87000000000]
+    (300a, 0084) Beam Dose                           DS: "1.0275401"
+    (300a, 0086) Beam Meterset                       DS: "116.0036697"
+    (300c, 0006) Referenced Beam Number              IS: "1"
+    ---------]
+
+The 'quiet' argument shows a minimal version of some of the information in the
+file, using just the DICOM keyword and value (not showing the tag numbers
+and VR). The example below shows the quiet mode with an image slice::
+
+    pydicom show -q pydicom::ct_small.dcm
+
+    SOPClassUID: CT Image Storage
+    PatientName: CompressedSamples^CT1
+    PatientID: 1CT1
+    StudyID: 1CT1
+    StudyDate: 20040119
+    StudyTime: 072730
+    StudyDescription: e+1
+    Image: 16-bit CT 128x128 pixels Slice location: -77.2040634155
+
+And the following example shows an RT Plan in quiet mode::
+
+    pydicom show -q pydicom::rtplan.dcm
+
+    SOPClassUID: RT Plan Storage
+    PatientName: Last^First^mid^pre
+    PatientID: id00001
+    StudyID: study1
+    StudyDate: 20030716
+    StudyTime: 153557
+    StudyDescription: N/A
+    Plan Label: Plan1  Plan Name: Plan1
+    Fraction Group 1  30 fraction(s) planned
+    Brachy Application Setups: 0
+    Beam 1 Dose 1.02754010000000 Meterset 116.003669700000
+    Beam 1 'Field 1' TREATMENT STATIC PHOTON energy 6.00000000000000 gantry 0.0, coll 0.0, couch 0.0 (0 wedges, 0 comps, 0 boli, 0 blocks)
+
+All quiet modes show the SOP Class UID, patient and study information as shown
+in the above two examples. After that are custom values for different classes.
+Currently "Image Storage" and "RT Plan Storage" classes have custom extra
+information.  Please submit an issue on the *pydicom* issues list (or even
```

Comment:
Maybe remove the "submit as issue" and just leave the PR?

Reply:
I like the idea of getting peoples' ideas even if there are not able or willing to code it themselves.  But I can write more simply as just 'submit an issue or a pull request'.

Reply:
Fair enough